src/gluonts/mx/model/estimator.py

```diff
@@ -39,7 +40,7 @@ class TrainOutput(NamedTuple):
     predictor: Predictor
 
 
-class GluonEstimator(Estimator):
+class GluonEstimator(Estimator, IncrementallyTrainable):
```

Comment:
I don't think we want to make this dependency explicit. I remember we had issues with the `PandasDataset` deriving from `Dataset`.